modules/service/src/main/scala/lucuma/odb/service/ConfigurationService.scala

```diff
@@ -249,18 +249,18 @@ object ConfigurationService {
           crs <- hc.downFields("configurationRequests", "matches").as[List[ConfigurationRequest]]
         yield (id, crs)
 
-      private given Decoder[((Program.Id, Observation.Id), Configuration)] = hc =>
+      private given Decoder[((Program.Id, Observation.Id), Option[Configuration])] = hc =>
         for
           pid <- hc.downFields("program", "id").as[Program.Id]
           oid <- hc.downField("id").as[Observation.Id]
-          cfg <- hc.downField("configuration").as[Configuration]
+          cfg  = hc.downField("configuration").as[Configuration].toOption
```

Comment:
The issue here is that some observations don't have a valid configuration and I need to filter these out.